kibana-reports/server/model/index.ts

```diff
@@ -61,62 +64,100 @@ export const cronSchema = schema.object({
   }),
 });
 
-export const emailSchema = schema.object({
-  subject: schema.string(),
-  body: schema.string(),
-  has_attachment: schema.boolean({ defaultValue: true }),
-  recipients: schema.arrayOf(schema.string(), { minSize: 1 }),
-});
-
-export const reportSchema = schema.object({
-  report_name: schema.string(),
-  report_source: schema.oneOf([
-    schema.literal('Dashboard'),
-    schema.literal('Visualization'),
-    schema.literal('Saved search'),
-  ]),
-  report_type: schema.oneOf([
-    schema.literal('Download'),
-    schema.literal('Alert'),
-    schema.literal('Schedule'),
+export const scheduleSchema = schema.object({
+  schedule_type: schema.oneOf([
+    /*
+    TODO: Alerting will be added in the future.
+    Currently @kbn/config-schema has no support for more than 2 conditions, keep an eye on library update
+    */
+    // schema.literal('Future Date'),
```

Comment:
Can we remove this line and add it in the comment above instead that Future date is not supported?

Reply:
sure, will do